lib/toxiproxy.rb

```diff
@@ -199,8 +203,12 @@ def toxics(direction)
 
   private
 
+  def self.uri
+    @uri ||= ::URI.parse(ENV.fetch('TOXIPROXY_HOST', 'http://127.0.0.1:8474'))
```

Comment:
Using environment variables is extremely uncommon in Ruby. Especially for libraries. I don't think we should support this in Toxiproxy core. Instead, set this where you use the library.